woodwork/serializers/parquet_serializer.py

```diff
@@ -12,6 +21,10 @@ class ParquetSerializer(Serializer):
 
     format = "parquet"
 
+    def __init__(self, path, filename, data_subdirectory, typing_info_filename):
+        super().__init__(path, filename, data_subdirectory, typing_info_filename)
+        self.typing_info_filename = None
```

Comment:
should we warn the user that we're ignoring the`typing_info_filename` if they provided one?

Reply:
Sure. Added [82256d9](https://github.com/alteryx/woodwork/pull/1377/commits/82256d90c00a47c4c2e53cc16d292d90e267ba19)

Reply:
Updated to check here ([e1cec8a](https://github.com/alteryx/woodwork/pull/1377/commits/e1cec8ae45bbecca7534f88832e19bdb26d59d1d)) to account for the fact that the default parameter value will be `woodwork_typing_info.json` when the user doesn't pass a value. This means that if the user actually specifies `woodwork_typing_info.json` they won't get a warning, but if they specify any other value except `None` they will get a warning.